src/poetry/console/commands/init.py

```diff
@@ -91,6 +93,7 @@ def handle(self) -> int:
                 )
                 return 1
 
+        config = Config()
```

Comment:
Updated to read the global config, instead of adding a new flag to `init` (https://github.com/python-poetry/poetry/pull/3996#discussion_r851708290)